packages/kbn-ftr-common-functional-services/services/index_patterns.ts

```diff
@@ -6,10 +6,14 @@
  * Side Public License, v 1.
  */
 
-import { ELASTIC_HTTP_VERSION_HEADER } from '@kbn/core-http-common';
-import { INITIAL_REST_VERSION } from '@kbn/data-views-plugin/server/constants';
-import { DataViewSpec } from '@kbn/data-plugin/common';
-import { FtrService } from '../ftr_provider_context';
+import type { DataViewSpec } from '@kbn/data-plugin/common';
+import { FtrService } from './ftr_provider_context';
+
+// The fallowing two constant values live in
+// '@kbn/data-views-plugin/server/constants' and
+// '@kbn/core-http-common' respectively.
+const INITIAL_REST_VERSION = '2023-10-31' as const;
+const ELASTIC_HTTP_VERSION_HEADER = 'elastic-api-version' as const;
```

Comment:
_question_: Any reasons not to import it from original packages?

Reply:
I think I left that as a placeholder but failed to add a "TODO" comment. 
Changed it back and running the type checker locally now, after I ran ` node scripts/lint_ts_projects --fix`